WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.wordpress.android.ui.people.utils;
+
+import com.android.volley.VolleyError;
+import com.wordpress.rest.RestRequest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.AppLog.T;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PeopleUtils {
+
+    public static void fetchUsers(String siteID, final int localTableBlogId, final PeopleUtils.Callback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null) {
+                    try {
+                        JSONArray jsonArray = jsonObject.getJSONArray("users");
+                        List<Person> people = peopleListFromJSON(jsonArray, localTableBlogId);
+                        PeopleTable.savePeople(people);
+
+                        if (callback != null) {
+                            callback.onSuccess();
+                        }
+                    }
+                    catch (JSONException e) {
```

Comment:
Exception is caught but no form of callback is called. This can lead to the caller get stuck waiting for the procedure to finish. Perhaps an extra callback function can be added or the existing `onError()` be reused.